x-pack/plugins/inference/common/chat_complete/without_chunk_events.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { filter, OperatorFunction } from 'rxjs';
+import { ChatCompletionChunkEvent, ChatCompletionEvent, ChatCompletionEventType } from '.';
+
+export function withoutChunkEvents<T extends ChatCompletionEvent>(): OperatorFunction<
```

Comment:
At some point perhaps these could be exported from chat_complete/index.ts or some place else for easier access if we see them as common utility functions engineers would need.